plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java

```diff
@@ -74,6 +78,9 @@ public void execute(Player p, @Optional String targetGroup, String minX, String
 						continue;
 					}
 					Block protecting = ReinforcementLogic.getResponsibleBlock(current);
+					if (skipReins) {
+						continue;
+					}
```

Comment:
Doesn't this just skip all reinforcements?

Reply:
No? We do multiple checks on the block first i.e if its air etc before creating the reinforcement

Reply:
my bad, good catch